frame/instrumental-strategy-pablo/src/mock/runtime.rs

```diff
@@ -14,6 +14,7 @@ pub type Amount = i128;
 pub type BlockNumber = u64;
 pub type Balance = u128;
 pub type CurrencyId = u128;
+pub type PoolId = u128;
```

Comment:
Maybe changes for this file from another PR?.. They were not in the previous commits.